code/app/build.gradle

```diff
@@ -9,7 +9,7 @@ android {
 
     defaultConfig {
         applicationId "com.cmput301w21t06.crowdfly"
-        minSdkVersion 23
```

Comment:
Is this needed or an accident?

Reply:
@Jinlehs Agreed, can we keep the api version 23? If not can you update the ci/cd script to use the updated version as well?

Reply:
sdk version should be fine at version 23.

Reply:
@Jinlehs Can you change it and commit that change?